docs/src/search.md

```diff
@@ -30,55 +30,81 @@ the vector column and compute the distance.
 
 === "Python"
 
-    ```python
-    import lancedb
-    db = lancedb.connect("data/sample-lancedb")
+<!--python 
+import lancedb
+import numpy as np
+uri = "data/sample-lancedb"
+db = lancedb.connect(uri)
 
-    tbl = db.open_table("my_vectors")
+data = [{"vector": row, "item": f"item {i}"}
+     for i, row in enumerate(np.random.random((10_000, 1536)).astype('float32'))]
 
-    df = tbl.search(np.random.random((768)))
-        .limit(10)
-        .to_df()
-    ```
+db.create_table("my_vectors", data=data)
+-->
+```python
+import lancedb
+import numpy as np
+
+db = lancedb.connect("data/sample-lancedb")
+
+tbl = db.open_table("my_vectors")
+
+df_1 = tbl.search(np.random.random((1536))) \
+    .limit(10) \
+    .to_df()
+```
 
 === "JavaScript"
 
-    ```javascript
-    const vectordb = require('vectordb')
-    const db = await vectordb.connect('data/sample-lancedb')
+<!--javascript 
```

Comment:
I was thinking that we should exclude just the metricType section for now instead of the entire file. Do you think its better to just exclude the entire file?